test/addons/dlopen-ping-pong/binding.cc

```diff
@@ -5,6 +5,7 @@
 
 #include <dlfcn.h>
 
+__attribute__((visibility("default")))
```

Comment:
Why not simply `NODE_EXTERN`?

Reply:
It does the opposite on Windows (export rather than import.) That wouldn't matter here since it's a unix-only test but it matters in the openssl addon tests.